backend/src/sockets/disconnect.js

```diff
@@ -1,9 +1,18 @@
+import games from '../domain/Games';
 /**
  * Socket event for when a user disconnects.
  * @param {*} socket
  */
 export function disconnect(socket) {
   socket.on('disconnect', () => {
+    const socketid = socket.id;
+    const activeGames = games.getGames();
+    activeGames.forEach(([gameid, game]) => {
+      game.session.users.filter((userid) => userid != socketid);
+      if (game.session.users.size == 0) {
+        games.removeGame(gameid);
```

Comment:
This is illegal

Reply:
Can find the room the user is currently in, and then just eject that room, if no users